### PR TITLE
WIP: Add quickstart devs

### PR DESCRIPTION
**Pull request by HereThereBeDragons**
Related https://github.com/cvmfs/cvmfs/issues/3044

Please give feedback

Todos:
- Add references to the different topics within the documentation
- Unify/Streamline layout (e.g. what type of header is used where)

### CODE REVIEW

File: devs-cpt-git.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+The name of the branch should be self-exanatory with a prefix (listed below) that describes the general nature of the work done within the branch, e.g. ``fix-openssl3-workaround``.
+It might be useful to add the issue number as a postfix to the branch name, e.g. ``fix-catalogXattr#2900``.
+
+=========== =================
+Type        Prefix
+=========== =================
+Fix         ``fix-``
+Feature     ``feature-``
+JIRA Ticket ``CVM-<Number>``
```
Comment (jblomer) [inline devs-cpt-git.rst:31]: Perhaps we can extend that for github, like `gh-<Number>`

**Comment by jblomer** [comment]
That's pretty cool!  I think we should put this documentation as markdown in the main sources under `/doc`.  But we should merge here your fixes to the quickstart guide and the authors list.

**Comment by HereThereBeDragons** [comment]
ok, should be no problem to create this also as markdowns. the original doc was done in markdown anyway.

btw one more thing i forgot: the file structure
i have now put every section in a different file - which means the entire chapter cannot be viewed as continuous scrolling experience.
if we want this, i have to reconsider how to solve it. for editing the separate files are definitely better.

File: _static/css/custom.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+/* and fix wrap bug per https://rackerlabs.github.io/docs-rackspace/tools/rtd-tables.html */
+.wy-table-responsive table td {
+    /* !important prevents the common CSS stylesheets from overriding
+    this as on RTD they are loaded after this stylesheet */
+    white-space: normal !important;
+}
+
+.wy-table-responsive {
+    overflow: visible !important;
+}
```
Comment (jblomer) [inline _static/css/custom.css:16]: Missing newline at the end of the file

File: cpt-configure.rst [hunk 3]
```diff
@@ -1173,7 +1173,7 @@ The ``cvmfs_talk`` command provides a way to control a currently running
 CernVM-FS process and to extract information about the status of the
 corresponding mount point. Most of the commands are for special purposes
 only or covered by more convenient commands, such as
-``cvmfs_config showconfig`` or ``cvmfs_config stat``. Three commands might
+``cvmfs_config showconfig`` or ``cvmfs_config stat``. Four commands might
```
Comment (jblomer) [inline cpt-configure.rst:1176]:
```suggestion
``cvmfs_config showconfig`` or ``cvmfs_config stat``. The following commands might
```

**Review by jblomer — CHANGES_REQUESTED** [review]
Many thanks! Almost ready to be merged.

**Comment by HereThereBeDragons** [comment]
Close, because it is now in cvmfs main repo

**Pull request closed without merging**